src/analysis_file.py

```diff
@@ -152,15 +151,19 @@ def generate(user, data, csv_by_message_output_path, csv_by_individual_output_pa
         export_keys.extend(matrix_keys)
         export_keys.extend(binary_keys)
         export_keys.extend(concat_keys)
-        export_keys.extend(demog_keys)
+        export_keys.extend(demog_and_follow_up_keys)
         
-        # Set consent withdrawn based on presence of data coded as "stop"
+        # Set consent withdrawn based on presence of demog data coded as "stop"
         ConsentUtils.determine_consent_withdrawn(
             user, data, PipelineConfiguration.DEMOGS_CODING_PLANS, consent_withdrawn_key)
```

Comment:
I think you could simplify this to 
```suggestion
            user, data, PipelineConfiguration.DEMOGS_CODING_PLANS + PipelineConfiguration.FOLLOW_UP_CODING_PLANS, consent_withdrawn_key)
```